packages/grid/src/vaadin-grid-sorter.js

```diff
@@ -237,6 +240,7 @@ class GridSorter extends ThemableMixin(DirMixin(PolymerElement)) {
 
     e.preventDefault();
     this._shiftClick = e.shiftKey;
+    this._fromSorterClick = true;
```

Comment:
`_shiftClick` seems to have been defined in the `properties` object. Would make sense to align `_fromSorterClick`. I'd be in favor of having neither of the properties listed there.

Reply:
Removed the private property. 

Since it doesn't have a default value anymore, I added a nullish coalescing operator to make sure that the properties on the detail have the right types. Could've also added a default value on the `ready` method, not sure which one is best.

Reply:
I think the latest changes haven't been pushed

Reply:
Pushed now 😅